packages/vue/src/examples/pages/home/Home.vue

```diff
@@ -209,157 +209,269 @@ export default {
         },
         {
           title: "Cream Beach Bag",
           image: "assets/storybook/Home/productA.jpg",
-          price: { regular: "50.00 $" },
+          price: { regular: "$ 50.00 " },
           rating: { max: 5, score: 4 },
-          isOnWishlist: false
+          isOnWishlist: false,
+          reviews: 8,
+          badgeLabel: "",
+          badgeColor: "color-primary"
         },
         {
           title: "Cream Beach Bag",
           image: "assets/storybook/Home/productB.jpg",
-          price: { regular: "50.00 $" },
+          price: { regular: "$ 50.00 " },
           rating: { max: 5, score: 4 },
-          isOnWishlist: false
+          isOnWishlist: false,
+          reviews: 8,
+          badgeLabel: "",
+          badgeColor: "color-primary"
         }
-      ]
+      ],
+      sectionLevel: 3,
+      instagramFeed: {
+        imageA: {
+          mobile: {
+            url: "/assets/storybook/Home/imageA.png",
+            content: "angelina_trn"
+          },
+          desktop: {
+            url: "/assets/storybook/Home/imageA.png",
+            content: "@instagram"
+          }
+        },
+        imageB: {
+          mobile: {
+            url: "/assets/storybook/Home/imageB.png",
+            content: "angelina_trn"
+          },
+          desktop: {
+            url: "/assets/storybook/Home/imageB_top.png",
+            content: "@instagram"
+          }
+        },
+        imageC: {
+          mobile: {
+            url: "/assets/storybook/Home/imageC.jpg",
+            content: "angelina_trn"
+          },
+          desktop: {
+            url: "assets/storybook/Home/imageA_bottom.png",
+            content: "@instagram"
+          }
+        },
+        imageD: {
+          mobile: {
+            url: "/assets/storybook/Home/imageD.jpg",
+            content: "angelina_trn"
+          },
+          desktop: {
+            url: "/assets/storybook/Home/imageB.png",
+            content: "@instagram"
+          }
+        }
+      }
     };
   },
   methods: {
     toggleWishlist(index) {
-      this.products[index].isOnWishlist = !this.products[index].isOnWishlist;
+      return (this.products[index].isOnWishlist = !this.products[index]
+        .isOnWishlist);
     }
   }
 };
 </script>
 <style lang="scss" scoped>
 @import "~@storefront-ui/vue/styles";
-#home {
+.home {
+  display: flex;
+  flex-direction: column;
   box-sizing: border-box;
-  @include for-desktop {
-    max-width: 1240px;
-    margin: 0 auto;
-  }
-}
-.section {
-  padding: 0 var(--spacer-big);
-  @include for-desktop {
-    padding: 0;
-  }
-}
-.banner-grid {
-  margin: var(--spacer-big) 0;
-  @include for-desktop {
-    margin: var(--spacer-extra-big) 0;
+  padding: 0 var(--spacer-sm);
+  &__hero {
+    display: flex;
+    order: 1;
+    margin: var(--spacer-xl) 0 0 0;
   }
-}
-.banner-central {
-  @include for-desktop {
-    --banner-container-flex: 0 0 70%;
+  &__heading {
+    display: flex;
+    order: 2;
```

Comment:
I think we don't need flex for whole page and order properties;)

Reply:
@pspaczek  should I use sth else to change order of components between mobile and desktop? or the order should be the same (from desktop or mobile)?